src/delegation/EscrowIVotesAdapter.sol

```diff
@@ -379,11 +379,14 @@ contract EscrowIVotesAdapter is
                 _checkpoint(bias, slope, fromDelegatee);
             }
 
-            numberOfDelegatedTokens[_from]--;
+            if (tokenIsDelegated(_tokenId)) {
+                numberOfDelegatedTokens[_from]--;
 
-            emit TokensUndelegated(_from, fromDelegatee, tokenIds);
+                emit TokensUndelegated(_from, fromDelegatee, tokenIds);
```

Comment:
this emits an event for an array of token ids under the condition that a single tokenId is delegated, is this expected?